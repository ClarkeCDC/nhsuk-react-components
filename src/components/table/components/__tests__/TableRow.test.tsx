import { mount } from 'enzyme';
import React from 'react';
import TableRow from '../TableRow';

describe('Table.Row', () => {
  it('matches snapshot', () => {
    const wrapper = mount(
      <table>
        <TableRow />
      </table>,
    );
    expect(wrapper).toMatchSnapshot();
    wrapper.unmount();
  });
});
