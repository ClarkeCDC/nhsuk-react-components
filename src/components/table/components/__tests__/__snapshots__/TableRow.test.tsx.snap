// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Table.Row matches snapshot 1`] = `
<table>
  <Table.Row>
    <tr
      className="nhsuk-table__row"
    />
  </Table.Row>
</table>
`;
